### ISSUE TITLE
unexpected keyword argument 'springtabitem'

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#625]
_[Trac time 20090513 163540Z]_ Email bug report:

```
#---- System Information ----#
Peppy Version: 0.9.27
Operating System: Linux 2.6.24-24-generic i686
Python Version: 2.5.2 (c0646b4d9673fcefcf99d85a109fd7d45bef1f69:60911, Jul 31 2008, 17:28:52)
[GCC 4.2.3 (Ubuntu 4.2.3-2ubuntu7)]
wxPython Version: 2.8.7.1 (gtk2-unicode)
wxPython Info: (__WXGTK__, wxGTK, unicode, gtk2, wx-assertions-off, SWIG-1.3.29)
Python Encoding: Default=ascii  File=UTF-8
wxPython Encoding: latin1
System Architecture: 32bit i686
Byte order: little
Frozen: False
#---- End System Information ----#

#---- Traceback Info ----#
*** Wed May 13 11:10:23 2009 ***
Traceback (most recent call last):
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 238, in OnLeftDown
  self.GetParent().setRadio(self)
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 354, in setRadio
  self.popupItem(item)
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 365, in popupItem
  self._tab_renderer.showPopup(self, item)
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 160, in showPopup
  popup, child = item.getPopup()
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 256, in getPopup
  self.window_cb(self.popup, springtabitem=self, **self.kwargs)
TypeError: __init__() got an unexpected keyword argument 'springtabitem'


*** Wed May 13 11:10:26 2009 ***
Traceback (most recent call last):
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/major.py", line 845, in OnFocus
  self.frame.spring.clearRadio()
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 373, in clearRadio
  self._tab_renderer.showPopup(self, self._radio, False)
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 160, in showPopup
  popup, child = item.getPopup()
 File "/usr/lib/python2.5/site-packages/peppy-0.9.27-py2.5.egg/peppy/lib/springtabs.py", line 257, in getPopup
  child = self.popup.GetChildren()[0]
 File "/usr/lib/python2.5/site-packages/wx-2.8-gtk2-unicode/wx/_core.py", line 8214, in __getitem__
  return _core_.WindowList___getitem__(*args, **kwargs)
IndexError: sequence index out of range
#---- End Traceback Info ----#
```

**Comment by robmcmullen** [comment]
_[Trac time 20090514 174248Z]_ 47681189c12dec7fa39fbb1ec55c5ed37561b28f Fixed #625: removed unnecessary (?) keyword arguments from springtab callback
- moved window_cb into argument list instead of kwargs -- was that necessary before???
- changed ErrorLogSidebar to construct the wx.Window first instead of the sidebar due to comments in bug #566

**Comment by robmcmullen** [comment]
_[Trac time 20090514 181805Z]_ 407fcd9fd20f36d9cbaad1dba50711e1f32af40c Refs #625: fixed incomplete merge from multikey branch
